### PR TITLE
fix:Add protection against

### PR DESCRIPTION
**Pull request by Pantom-007**
## Ticket
#3

### CODE REVIEW

File: contracts/Rsp.sol [hunk 1]
```diff
@@ -25,14 +25,14 @@ contract Rsp is Base, ERC20 {
...
+    function _checkResult(Hands player, Hands bot) internal pure returns(okay) {
         // draw
-        if (player == computer) { return okay.Draw; }
+        if (player == bot) { return okay.Draw; }
 
         // win
-        if (player == Hands.Rock && computer == Hands.Scissors) { return okay.Win; }
-        if (player == Hands.Paper && computer == Hands.Rock) { return okay.Win; }
-        if (player == Hands.Scissors && computer == Hands.Paper) { return okay.Win; }
+        if (player == Hands.Paper && bot == Hands.Rock) { return okay.Win; }
```
Comment (Kurama-nine) [inline contracts/Rsp.sol:33]: This is useful.
Thank you.

**Review by Kurama-nine — APPROVED** [review]
LGTM !

File: contracts/Rsp.sol [hunk 2]
```diff
@@ -25,14 +25,14 @@ contract Rsp is Base, ERC20 {
...
+        if (player == Hands.Scissors && bot == Hands.Paper) { return okay.Win; }
+        if (player == Hands.Rock && bot == Hands.Scissors) { return okay.Win; }
```
Comment (TelamonPegasus) [inline contracts/Rsp.sol:35]: In this case, how can we set the variable as initial?

**Review by TelamonPegasus — APPROVED** [review]
Thank you !

File: contracts/Rsp.sol (hunk 1 above)
Comment (Pantom-007) [reply contracts/Rsp.sol:33]: I`ve fixed about Add protection against reentrancy in Token
I`d like to fix next issue

File: contracts/Rsp.sol (hunk 2 above)
Comment (Pantom-007) [inline contracts/Rsp.sol:35]: It is already formatted.
what i have to do for your project?

**Review by Pantom-007 — COMMENTED** [review]
I`ve checked

**Pull request merged**